frontend/src/components/DestructionList/DestructionList.tsx

```diff
@@ -92,6 +94,19 @@ export function DestructionList({
     setSearchParams(activeParams);
   };
 
+  /**
+   * Gets called when the fields selection is changed.
+   * @param fields
+   */
+  const onFieldsChange = async (fields: TypedField[]) => {
+    const activeFields = fields.filter((f) => f.active !== false);
+    const inActiveFields = fields.filter((f) => f.active === false);
+    await addToFieldSelection(storageKey, activeFields);
+    await removeFromFieldSelection(storageKey, inActiveFields);
```

Comment:
The only reason I can think of is for consistency with the zaak selection lib.

The zaak select counterpart doesn't work on the full dataset (as it's a paginated endpoint. and therefore adds to and remove from but doesn't overwrite the data set as it might be populated with unknown data.

This lib simply behaves the same way for consistency.